src/renderer/pages/project/chat/graph.ts

```
@@ -66,7 +66,7 @@
             console: { after: true },
           },
           prompt: {
-            update: ":validateSchema.error",
+             update: ":nextPrompt.array",
```

ESLint

[error] Delete `·`

(prettier/prettier)